app-tracking-protection/vpn-impl/src/test/java/com/duckduckgo/mobile/android/vpn/feature/settings/NetworkSwitchSettingPluginTestView.kt

```diff
@@ -25,7 +25,7 @@ import org.junit.Before
 import org.junit.Test
 import org.mockito.kotlin.*
 
-class NetworkSwitchSettingPluginTest {
+class NetworkSwitchSettingPluginTestView {
```

Comment:
I see this was renamed from `NetworkSwitchSettingPluginTest` to `NetworkSwitchSettingPluginTestView`, but I think it should be `NetworkSwitchingSettingPluginTest` (third version :smiley:).